l5kit/l5kit/rasterization/stub_rasterizer.py

```diff
@@ -11,24 +12,18 @@ class StubRasterizer(Rasterizer):
     """
 
     def __init__(
-        self,
-        raster_size: Tuple[int, int],
-        pixel_size: np.ndarray,
-        ego_center: np.ndarray,
-        filter_agents_threshold: float,
+        self, render_context: RenderContext, filter_agents_threshold: float,
     ):
         """
 
-        Arguments:
-            raster_size {Tuple[int, int]} -- Desired output image size
-            pixel_size {np.ndarray} -- Dimensions of one pixel in the real world
-            ego_center {np.ndarray} -- Center of ego in the image, [0.5,0.5] would be in the image center.
-            filter_agents_threshold {float} -- Value between 0 and 1 used to filter uncertain agent detections
+        Args:
+            render_context (RenderContext): Render Context
+            filter_agents_threshold (float): Value between 0 and 1 used to filter uncertain agent detections
```

Comment:
Can you maybe provide more info on render context? You use this quite a lot but  I haven't seen an explanation. Like saying it consists of bla and blub because blabla

Reply:
I'll do my best but I was not the one who added it actually :)